src/main/java/appeng/tile/storage/TileIOPort.java

```diff
@@ -115,19 +103,22 @@ public TileIOPort()
 		this.manager.registerSetting( Settings.REDSTONE_CONTROLLED, RedstoneMode.IGNORE );
 		this.manager.registerSetting( Settings.FULLNESS_MODE, FullnessMode.EMPTY );
 		this.manager.registerSetting( Settings.OPERATION_MODE, OperationMode.EMPTY );
-		this.cells = new AppEngInternalInventory( this, 12 );
 		this.mySrc = new MachineSource( this );
 		this.lastRedstoneState = YesNo.UNDECIDED;
 
 		final Block ioPortBlock = AEApi.instance().definitions().blocks().iOPort().maybeBlock().get();
 		this.upgrades = new BlockUpgradeInventory( ioPortBlock, this, 3 );
+
+		inputCells.setFilter( AEItemFilters.INSERT_ONLY );
+		outputCells.setFilter( AEItemFilters.EXTRACT_ONLY );
 	}
 
 	@TileEvent( TileEventType.WORLD_NBT_WRITE )
 	public void writeToNBT_TileIOPort( final NBTTagCompound data )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)